content/docs/thinking-in-react.md

```diff
@@ -31,116 +31,116 @@ Our JSON API returns some data that looks like this:
 ];
 ```
 
-## Step 1: Break The UI Into A Component Hierarchy {#step-1-break-the-ui-into-a-component-hierarchy}
+## स्टेप १ : UI को कौम्पोनॅन्ट पदक्रम में तोड़े {#step-1-break-the-ui-into-a-component-hierarchy}
```

Comment:
`स्टेप १ : UI को कौम्पोनॅन्ट पदक्रम में तोड़े` => `स्टेप १: UI को कौम्पोनॅन्ट हायरार्की में ब्रेक करें`

Reply:
`स्टेप १` => `स्टेप 1`

let's keep numbers in english.
sorry missed this in previous review